pkg/kubernetes/views/imagestream-body.html

```diff
@@ -23,14 +23,25 @@
         <i title="Images only accessible to members" class="fa fa-lock imagestream-lock"></i>
       </div>
     <dt translatable="yes">Pull repository</dt>
-    <dd ng-if="stream.spec.dockerImageRepository"><tt>{{stream.spec.dockerImageRepository}}</tt></dd>
-    <dd ng-if="!stream.spec.dockerImageRepository"><em translatable="yes">None</em></dd>
+    <dd ng-if="stream.status.dockerImageRepository"><tt>{{stream.status.dockerImageRepository}}</tt></dd>
+    <dd ng-if="!stream.status.dockerImageRepository"><em translatable="yes">None</em></dd>
```

Comment:
This will list which repository is being pulled rather than the one that is requested to pull. Is there a reason for this? Should we make it conditional and list both if they differ? Interested in what specifically this fixes.